### PR TITLE
feat: improve sql codemirror, style and error management

### PR DESCRIPTION
**Pull request by preda-bogdan**
### Summary

- Moved the autocomplete key binding to `Shift + Spacebar` so that it works on all platforms the same.
- Changed the styles to fix issues with padding, multi-line query, and bad highlighting of built-in functions.
- Added a line to indicate available DB prefixes for the WordPress database.
- Fixed issue with errors not being reported if syntax errors were present. Now proper error should be displayed.
- Allow queries to end in `;` we strip them if present, this should allow users to write queries more easily.

### Will affect the visual aspect of the product

YES

### Screenshots 
<details open>
    <summary>Default Query and style changes</summary>

image
</details>

### Test instructions

1. On a fresh instance with Visualizer Pro enabled
2. Check the Database Source for a chart
3. Check the default query is working
4. Check that the shortcut `Shift + Spacebar` is recommending query suggestions.
5. Check that the db prefix is shown.
6. Check that the query can end in `;` or work without.
7. Check that query errors are displayed if a query is invalid.

## Check before Pull Request is ready:

* [x] I have [written a test](CONTRIBUTING.md#writing-an-acceptance-test) and included it in this PR
* [x] I have [run all tests](CONTRIBUTING.md#run-tests) and they pass
* [x] The code passes when [running the PHP CodeSniffer](CONTRIBUTING.md#run-php-codesniffer)
* [x] Code meets [WordPress Coding Standards](CONTRIBUTING.md#coding-standards) for PHP, HTML, CSS and JS
* [x] [Security and Sanitization](CONTRIBUTING.md#security-and-sanitization) requirements have been followed
* [x] I have assigned a reviewer or two to review this PR (if you're not sure who to assign, we can do this step for you)

Closes Codeinwp/visualizer-pro#448.

### CODE REVIEW

File: js/frame.js [hunk 1]
```diff
@@ -226,6 +226,7 @@
 
     // https://codemirror.net/
     function init_db_import_component(){
+        console.log( 'HERE' );
```
Comment (Soare-Robert-Daniel) [inline js/frame.js:229]: You can remove this log.

**Review by Soare-Robert-Daniel — APPROVED** [review]
Looking good 🔥 . Just a stray `console.log` left.

**Comment by ineagu** [comment]
@preda-bogdan can this easily result in a chart?

**Comment by preda-bogdan** [comment]
> @preda-bogdan can this easily result in a chart? 

@ineagu Yes, the current default query will easily display a chart without any change from the user.

**Comment by irinelenache** [comment]
@preda-bogdan Tested and here is what i found:

- [ ] The shortcut is `control + space` in my case (using macOS)
- [ ] There's an empty space in the text editor that makes the code harder to read https://vertis.d.pr/i/X08hBQ
- [ ] The word 'YEAR' in the query is not readable because of the low contrast https://vertis.d.pr/i/OlQgvA

**Comment by irinelenache** [comment]
@preda-bogdan Indeed, the issues were related to caching. I checked again and everything's fine on my end 🚀

**Pull request merged**